src/sections/Home/styles.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const VideoHome = styled.video`
```

Comment:
`export const VideoHome = styled.video`
    height:${({isMobile})=>isMobile && '100%'};
    width:${({isMobile})=>!isMobile && '100%'};
    opacity: 0.6;
    padding-top: ${({theme})=> theme.scale.base*6.5}px;  
`;`

this would help to make the video behave somehow differently depending on the devices the page is been rendered on

use the same custom hook from the header component and pass the `isMobile` prop onto the video component